### PR TITLE
Return service instance instead db instance for rds

### PR DESCRIPTION
**Pull request by h00130372**

### CODE REVIEW

File: pkg/services/rds/update.go [hunk 1]
```diff
@@ -42,6 +42,19 @@ func (b *RDSBroker) Update(instanceID string, details brokerapi.UpdateDetails, a
...
+	instance, err, serviceErr := SyncStatusWithService(b, instanceID, details.ServiceID, details.PlanID, ids.TargetID)
+
+	if err != nil || serviceErr != nil {
+		return brokerapi.UpdateServiceSpec{}, fmt.Errorf("sync status failed. error: %s, service error: %s", err, serviceErr)
+	}
+	if instance.Status != "ACTIVE" {
+		return brokerapi.UpdateServiceSpec{},
+			brokerapi.NewFailureResponse(
+				fmt.Errorf("Can only update rds instance in ACTIVE, but in: %s", instance.Status),
+				422, "Can only update rds instance in ACTIVE")
```
Comment (kiwik) [inline pkg/services/rds/update.go]: you can use `http.StatusUnprocessableEntity` instead of `422`, looks more reasonable.

Comment (h00130372) [reply pkg/services/rds/update.go]: Thank you

**Comment by liu-sheng** [comment]
According to the testing results, this looks OK, thanks.

**Comment by edisonxiang** [comment]
LGTM Thanks:)

**Pull request merged**